tests/conftest.py

```diff
@@ -7,7 +7,7 @@ def pytest_addoption(parser):
     parser.addoption(
         '--sqlalchemy_url',
         action='store',
-        default='sqlite:///:memory:',
+        default='postgresql://postgres:postgres@localhost/skosprovider_sqlalchemy',
```

Comment:
Any particular reason for this change? Not necessarily against it, just don't know why we do it.

Reply:
ahh, local testing. This didn't need to be committed. Travis was failing for postgres exclusively.